libraries/webkit2gtk4.1/webkit2gtk4.1.info

```diff
@@ -1,8 +1,8 @@
 PRGNAM="webkit2gtk4.1"
 VERSION="2.44.2"
 HOMEPAGE="https://www.webkitgtk.org/"
-DOWNLOAD="https://www.webkitgtk.org/releases/webkitgtk-2.44.2.tar.xz"
-MD5SUM="b61ff37133cabc9ef5d870b296e9694d"
+DOWNLOAD="https://www.webkitgtk.org/releases/webkitgtk-2.44.2.tar.xz https://gitlab.gnome.org/GNOME/vala/-/raw/main/vapi/webkit2gtk-4.1.vapi https://gitlab.gnome.org/GNOME/vala/-/raw/main/vapi/webkit2gtk-web-extension-4.1.vapi https://gitlab.gnome.org/GNOME/vala/-/raw/main/vapi/javascriptcoregtk-4.1.vapi"
```

Comment:
isn't this always pointing to latest master branch? this will cause invalid md5sum everytime they pushed something

Reply:
sorry yes, it does. I've redone it to use 0.56.17 release tag see PR #80